typings/rendition.d.ts

```diff
@@ -433,6 +433,10 @@ declare module 'rendition' {
 		// Optionally provide a key that should be used as a unique identifier for each row
 		rowKey?: keyof T;
 		tbodyPrefix?: JSX.Element | JSX.Element[];
+		// Highlights a row. This property requires that you have provided a
+		// `rowKey` property: the row with a `rowKey` property that matches this
+		// value is highlighted.
+		highlightedRow?: number | string;
```

Comment:
The type here should match the cast for identifier in isHighlighted, since they represent the same thing. I think the `any` there is actually right - your key could be absolutely anything, including objects or arrays or symbols or whatever.

Reply:
Maybe `T[keyof T]` even though this doesn't make it a lot better than `any`.